src/ess/reflectometry/conversions.py

```diff
@@ -149,6 +149,9 @@ def add_masks(
     wb: WavelengthBins,
     zlim: ZIndexLimits,
 ) -> MaskedData[RunType]:
+    da.masks["beam_divergence_too_large"] = sc.abs(
+        da.coords["angle_from_center_of_beam"]
+    ) > sc.scalar(0.7, unit='deg')
```

Comment:
Should the 0.7 be a parameter to the workflow?
If not, can we add a comment documenting where the value of 0.7 comes from?

Reply:
Good point